### PR TITLE
#167132886 create input field atom

### PR DESCRIPTION
**Pull request by mosinmiloluwa01**
#### What does this PR do?
This PR creates an input field atom

#### Description of task to be completed?
- write test for the feature
- create style component for input field
- create input field component

#### How should this be manually tested?
- run `npm install`
- run `npm run styleguide` on your console

#### Any background context you want to provide?
user must have node installed

#### What are the relevant pivotal tracker stories?
[167132886](https://www.pivotaltracker.com/story/show/167132886)

#### Screenshots (if appropriate)
Screenshot 2019-07-09 at 7 18 28 PM

### CODE REVIEW

**Review by akinyeleolat — APPROVED** [review]

**Review by Eazybee — APPROVED** [review]

File: src/styles/variables/border.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export const borderRadius = {
+  fullRadius: '100%',
+  partialRadius: '50%',
+  noRadius: '0%',
```
Comment (benfluleck) [inline src/styles/variables/border.js]: Do you need a value for this?

File: src/styles/variables/border.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  edgeRadius: '4rem',
```
Comment (benfluleck) [inline src/styles/variables/border.js:2]: Why is this 4rem why not keep it consistent?

File: src/styles/variables/border.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+};
+
+export const boxShadow = {
+  slightshadow: '0px 4px 1px rgba(0, 0, 0, 0.1)',
```
Comment (benfluleck) [inline src/styles/variables/border.js]: This varies from element to element I think it should be on the element that implements it

File: src/styles/variables/colorPalette.js [hunk 4]
```diff
@@ -9,3 +9,7 @@ export const textColors = {
   primary: '#B02091',
   grey: '#7A757E',
 };
+
+export const backgroundColors = {
+  lightPink: 'rgba(233, 218, 218, 0.5)',
```
Comment (benfluleck) [inline src/styles/variables/colorPalette.js]: Keep it consistent use hex colors ?

File: src/styles/variables/spacing.js [hunk 5]
```diff
@@ -1,7 +1,12 @@
 const spacing = {
   zero: '0rem',
+  xs: '0.9rem',
   sm: '1.6rem',
   md: '3.2rem',
+  profileWidth: '7rem',
```
Comment (benfluleck) [inline src/styles/variables/spacing.js]: Why do you have colors of the same value? This is not good practice

File: src/styles/variables/spacing.js [hunk 6]
```diff
@@ -1,7 +1,12 @@
...
+  profileHeight: '7rem',
+  fullWidth: '100%',
```
Comment (benfluleck) [inline src/styles/variables/spacing.js]: 100% width is spacing

File: src/components/UI/atoms/InputField/InputField.jsx [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+  borderRadius: PropTypes.oneOf(Object.keys(themeBorderRadius)),
+};
+
+InputField.defaultProps = {
+  padding: 'zero',
+  fontSize: 'normal',
+  color: 'primary',
+  content: 'false',
+  background: 'lightPink',
+  width: 'fullWidth',
```
Comment (benfluleck) [inline src/components/UI/atoms/InputField/InputField.jsx]: use diplay: block to replicate 100% full width

**Review by benfluleck — CHANGES_REQUESTED** [review]

File: src/components/UI/atoms/InputField/InputField.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+  padding="xs"
+></InputField>
+```
+
+Search Bar Input Field Default:
+
+```jsx
+<InputField
+  type="text"
+        name="text"
```
Comment (benfluleck) [inline src/components/UI/atoms/InputField/InputField.md]: Sort out this indentation pleasse.

**Pull request merged**